src/main/java/teammates/ui/controller/Action.java

```diff
@@ -176,7 +176,8 @@ protected AccountAttributes authenticateNotLoggedInUser(String email, String cou
         boolean isUnknownKey = student == null;
         boolean isARegisteredUser = !isUnknownKey && student.googleId != null && !student.googleId.isEmpty();
         boolean isMissingAdditionalAuthenticationInfo = email == null || courseId == null;
-        boolean isAuthenticationFailure = !isUnknownKey && (!student.email.equals(email) || !student.course.equals(courseId));
+        boolean isAuthenticationFailure = !isUnknownKey && (!student.email.equals(email)
+                                          || !student.course.equals(courseId));
```

Comment:
Maybe shift this to be in line with `[!]student.email...` since it isn't really on the same level as `!isUnknownKey`